### DISCUSSION TITLE
Welcome to pangea dnd Discussions!

### DISCUSSION THREAD

**Discussion by Xhale1** [hello-pangea/dnd#417] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by anhtruongngo1** [comment]
"**No stop drag when no active drag**"
I am using it with a drag and drop area with react-big-calendar library But it is having a problem.
When I add a drop area to the calendar, an error occurs as shown. When I drag an item and drop it, I am calling an api. If that api has not finished in time when I drag and drop another item, it reports an error. **"No stop drag when no active drag**"
![image](https://github.com/user-attachments/assets/27e1746c-60f7-4239-bfe5-f792380a3767)
